.babelrc-v9.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 5 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27036&commitId=11f50b4eb03a53a8bae9afa66b6fc7db14e1c9b6&env=prod)

✅ There was 2 screenshots added, 0 screenshots removed, 1040 screenshots unchanged, 0 screenshots with different dimensions and 3 screenshots with visible difference.


  <details><summary><b>unknown</b> 5 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ContextualMenu.With split button submenu.click.chromium.png|11| Changed|
|ContextualMenu.With split button submenu.default.chromium.png|9| Changed|
|ContextualMenu.With split button submenu.hover.chromium.png|11| Changed|
|Pivot - Overflow.Tabs - RTL.Narrow - Last tab selected.chromium.png|0| Added|
|Pivot - Overflow.Tabs - RTL.Narrow - Overflow menu.chromium.png|0| Added|

</details>


<div id="vrtCommentfluentuiv8"/>